jscomp/super_errors/super_typetexp.ml

```diff
@@ -159,7 +159,11 @@ let setup () =
   Location.register_error_of_exn
     (function
       | Typetexp.Error (loc, env, err) ->
-        Some (Location.error_of_printer loc (report_error env) err)
+        Some (Location.error_of_printer loc (fun ppf ->
```

Comment:
so you mean 
`bsc -bs-super-error -c xx.re ` will not work but `bsc -c xx.re -bs-super-error` will be good?